src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -98,24 +98,12 @@ function WorkspaceTagsPage({policyTags, route}: WorkspaceTagsPageProps) {
                             errors: tag.errors ?? undefined,
                             enabled: tag.enabled,
                             isDisabled,
-                            rightElement: (
-                                <View style={styles.flexRow}>
-                                    <Text style={[styles.textSupporting, styles.alignSelfCenter, styles.pl2, styles.label]}>
-                                        {tag.enabled ? translate('workspace.common.enabled') : translate('workspace.common.disabled')}
-                                    </Text>
-                                    <View style={[styles.p1, styles.pl2]}>
-                                        <Icon
-                                            src={Expensicons.ArrowRight}
-                                            fill={theme.icon}
-                                        />
-                                    </View>
-                                </View>
-                            ),
+                            rightElement: <ListItemRightElementWithArrow enabled={tag.enabled} />,
                         };
                     }),
                 )
                 .flat(),
-        [policyTagLists, selectedTags, styles.alignSelfCenter, styles.flexRow, styles.label, styles.p1, styles.pl2, styles.textSupporting, theme.icon, translate],
+        [policyTagLists, selectedTags, translate],
```

Comment:
`translate` is unnecessary.
```suggestion
        [policyTagLists, selectedTags],
```